charts/cluster-scanner/values.yaml

```diff
@@ -0,0 +1,124 @@
+#basic Default values for cluster-scanner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  sysdigHost: ""
+  sysdigAccessKey: ""
+  clusterName: ""
+  sysdigSkipVerifyCertificate: false
+
+  scannerMode: "local" # or "multi"
+  loggingLevel: "INFO"
+
+eveEnabled: false
+eveIntegrationEnabled: false
+rootNamespace: "kube-system"
+
+multicluster:
+  # kubeconfigSecretName: ""
+
+localcluster:
+  # restrict access to specific docker secrets to obtain pull credentials. Optional.
+  # Default behaviour is listing all secrets
+  # rbac:
+  #   allowedPullSecrets:
+  #     namespaceOneName:
+  #       - myDockerSecretOne
+  #       - myDockerSecretTwo
+  #     anotherNamespace:
+  #       - myOtherDockerSecretOne
+  #       - myOtherDockerSecretTwo
+
+replicaCount: 2
+
+# Params to manage leader election of rsi component.
+# Leader election is implemented leveraging the native capabilities of Kubernetes
+# see: https://kubernetes.io/blog/2016/01/simple-leader-election-with-kubernetes/
+# leaderElectionLeaseNameOverride: ""
+# leaderElectionLeaseNamespaceOverride: ""
+
+rsi:
+  image:
+    repository: quay.io/sysdig/runtime-status-integrator
+    pullPolicy: IfNotPresent
+    tag: "0.1.2-vf589546"
+  ports:
+    metrics: 25000
+    probes: 7000
+  resources:
+    limits:
+      cpu: "1"
+      memory: 350Mi
+    requests:
+      cpu: "350m"
+      memory: 350Mi
+ise:
+  image:
+    repository: quay.io/sysdig/image-sbom-extractor
+    pullPolicy: IfNotPresent
+    tag: "0.1.2-vae4fc61"
+  ports:
+    metrics: 25001
+    probes: 7001
+  resources:
+    limits:
+      cpu: "1"
+      memory: 350Mi
+    requests:
+      cpu: "150m"
+      memory: 350Mi
+
+imagePullSecrets: []
```

Comment:
I suppose the image pull secret should be at component level (ise / rsi / ...)

Reply:
pullsecrets are defined once per pod, it's not at the container level